app/config/FrontendAppConfig.scala

```diff
@@ -40,6 +42,11 @@ class FrontendAppConfig @Inject() (configuration: Configuration) {
   lazy val loginContinueUrl: String = configuration.get[String]("urls.loginContinue")
   lazy val logoutUrl: String = configuration.get[String]("urls.logout")
 
+  lazy val estateHubUrl: String = configuration.get[String]("urls.estateHub")
```

Comment:
This is minor, but it seems like you would either want to use configuration.get[String] or loadConfig (maybe call it "loadString"?). It's a bit odd to have some doing it one way and some another.

Reply:
Replaced loadConfig